README.md

```diff
@@ -1,25 +1,29 @@
+[![Build Status](https://travis-ci.org/IBM/<repo>.svg?branch=master)](https://travis-ci.org/IBM/<repo>)
```

Comment:
this needs to be removed. it doesn't link to anything.

Reply:
@stevemart Lint and Travis is WIP 👍

Reply:
I figured :)

But we can add that line when linting and travis are actually up :)